cloudbuild.yaml

```diff
@@ -1,10 +1,10 @@
 # See https://cloud.google.com/cloud-build/docs/build-config
-timeout: 1200s
+timeout: 1800s
 options:
   substitution_option: ALLOW_LOOSE
   machineType: 'N1_HIGHCPU_8'
 steps:
-  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20200619-68869a4'
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20210331-c732583'
```

Comment:
This is a new image build here: https://github.com/kubernetes/test-infra/pull/21566
which now contains Golang 1.16

xref: https://github.com/kubernetes-sigs/cluster-api/issues/4383